win32/src/winapi/kernel32/dll.rs

```diff
@@ -200,16 +202,23 @@ pub fn load_library(machine: &mut Machine, filename: &str) -> HMODULE {
     }
 
     let mut contents = Vec::new();
-    {
-        let mut file = match machine.host.open(&filename, host::FileOptions::read()) {
+    let exe = machine.state.kernel32.cmdline.args.first().unwrap();
+    let exe_dir = std::path::Path::new(exe)
+        .parent()
+        .unwrap()
+        .to_string_lossy();
+    let dll_paths = [format!("{exe_dir}\\{filename}"), filename.to_string()];
```

Comment:
This is good, it totally removes the reason I added the --chdir flag haha

Reply:
chdir is still useful in combination with `cargo run`